.github/workflows/aws-ci.yaml

```diff
@@ -0,0 +1,107 @@
+name: CI tests AWS
+on:
+  pull_request_target:
+    types: [labeled, synchronize]  # Triggered when the PR is labeled or synchronized
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+jobs:
+  deploy-ec2:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Remove ok-to-test label if new commit
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const label = "ok-to-test";
+            const pr = context.payload.pull_request;
+            const labels = pr.labels.map(l => l.name);
+            if (labels.includes(label)) {
```

Comment:
this will remove ok-to-test for any label added by user?

Reply:
Add logic to remove ok-to-test only when the event is synchronize (i.e., new commit push), not when any label is added.